src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -101,6 +101,7 @@ function BaseSelectionList<TItem extends ListItem>(
         onLongPressRow,
         shouldShowTextInput = !!textInputLabel || !!textInputIconLeft,
         shouldShowListEmptyContent = true,
+        shouldIgnoreFocus,
```

Comment:
```suggestion
        shouldIgnoreFocus = false,
```
NAB: Pleasing the linter with passing a default value here instead of double negating the possible `undefined` seems to be more inline with usual practices followed in our codebase.

Reply:
Done